pkg/trace/fileserver.go

```diff
@@ -259,7 +259,9 @@ func (lt *LocalTracer) PushAll() error {
 
 // S3Download downloads files that match some prefix from an S3 bucket to a
 // local directory dst.
-func S3Download(dst, prefix string, cfg S3Config) error {
+// tableNames is a list of traced jsonl file names to download. If it is empty, all traces are downloaded.
+// tableNames should not have .jsonl suffix.
+func S3Download(dst, prefix string, fileNames []string, cfg S3Config) error {
```

Comment:
[nit] the comment name doesn't match the argument name. 

```suggestion
// fileNames is a list of traced jsonl file names to download. If it is empty, all traces are downloaded.
// fileNames should not have .jsonl suffix.
func S3Download(dst, prefix string, fileNames []string, cfg S3Config) error {
```

Reply:
Right, fixed https://github.com/celestiaorg/celestia-core/pull/1374/commits/bbae134e0a8d67e5d706fcb1908dcf51461f45fd